ex2-solution/src/components/app/index.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+
+import { Container, Header, Footer, Copyright } from "../../components/layout";
```

Comment:
i would prefer `../layout` or better `components/layout` which needs the modules root set